src/infra/cache/permissions.ts

```diff
@@ -0,0 +1,11 @@
+import type { permissions } from '@balena/pinejs';
```

Comment:
Can you rename this file `permissions-normalizer` please, when I saw the `cache/permissions` I assumed some form of permissions cache